install/upgrade/versions/1.7.1.sh

```diff
@@ -15,13 +17,23 @@
 ####### You can use \n within the string to create new lines.                   #######
 #######################################################################################
 
-upgrade_config_set_value 'UPGRADE_UPDATE_WEB_TEMPLATES' 'no'
-upgrade_config_set_value 'UPGRADE_UPDATE_DNS_TEMPLATES' 'no'
-upgrade_config_set_value 'UPGRADE_UPDATE_MAIL_TEMPLATES' 'no'
-upgrade_config_set_value 'UPGRADE_REBUILD_USERS' 'no'
+upgrade_config_set_value 'UPGRADE_UPDATE_WEB_TEMPLATES' 'false'
+upgrade_config_set_value 'UPGRADE_UPDATE_DNS_TEMPLATES' 'false'
+upgrade_config_set_value 'UPGRADE_UPDATE_MAIL_TEMPLATES' 'false'
+upgrade_config_set_value 'UPGRADE_REBUILD_USERS' 'false'
 upgrade_config_set_value 'UPGRADE_UPDATE_FILEMANAGER_CONFIG' 'false'
 
-grep 'v-change-user-password' $HESTIA/data/api/billing
-if [ $? -ne 0 ]; then
-	sed -i "s/v-make-tmp-file'/v-make-tmp-file,v-change-user-password'/g" $HESTIA/data/api/billing
+# Moved from 1.6.15.sh
```

Comment:
Even 1.6.15 wasn't release as such version If installed from 1.6.14 -> 1.6.15 still got installed so it should be fine.

Reply:
I know, but it's kind of weird to have an upgrade script for an unreleased version.

So move it without affecting users who manually update Hestia in the future.